source/ranges.tex

```diff
@@ -9341,6 +9357,550 @@
+@\exposid{pos_}@ -= n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator==}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr bool operator==(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} == y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator<=>}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr strong_ordering operator<=>(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} <=> y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(const @\exposid{iterator}@& x, difference_type y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = x;
+temp += y;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(difference_type x, const @\exposid{iterator}@& y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return y + x;}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator-(const @\exposid{iterator}@& x, difference_type y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = x;
+temp -= y;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr difference_type operator-(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} - y.\exposid{pos_};}
+\end{itemdescr}
+
+\rSec3[range.enumerate.sentinel]{Class template \tcode{enumerate_view::\exposid{sentinel}}}
+
+\indexlibraryglobal{enumerate_view::\exposid{sentinel}}%
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{view}@ V>
+    requires @\exposconcept{range-with-movable-references}@<V>
+  template<bool Const>
+  class enumerate_view<V>::@\exposidnc{sentinel}@ {                           // \expos
+    using @\exposidnc{Base}@ = @\exposidnc{maybe-const}@<Const, V>;                         // \expos
+    sentinel_t<@\exposidnc{Base}@> @\exposidnc{end_}@ = sentinel_t<@\exposidnc{Base}@>();                 // \expos
+    constexpr explicit @\exposid{sentinel}@(sentinel_t<@\exposid{Base}@> end);
```

Comment:
```suggestion
    constexpr explicit @\exposidnc{sentinel}@(sentinel_t<@\exposidnc{Base}@> end);          // \expos
```
Not in the paper.

Reply:
Done as a separate commit, but what about the other ctors for sentinel? 

If I had to guess, I would think the `\expos` would go on the line of the class definition (as was done originally n the paper), i.e. I would have thought that the [range.enumerate.iterator] synopsis should have been changed to match [range.enumerate.sentinel].

Reply:
See #4829.